packages/ui/src/components/Activity/ActivityScreenView.tsx

```diff
@@ -78,6 +83,14 @@ export function ActivityScreenView({
             console.warn('No parent found for reply', event);
           }
           break;
+        case 'group-ask':
+          if (event.group) {
+            markGroupRead(event.group);
```

Comment:
if we're only going to run both maybe pass a single handler that worries about that itself